### DISCUSSION TITLE
Usage of ram and cpu not showing correctly

### DISCUSSION THREAD

**Discussion by RayCodeCSC** [vilhelmprytz/pterodactyl-installer#305] [Q&A]
Hello, i have installed pterodactyl using your (script?). But some features not working correctly, like showing correct usage etc... Is still on 0.

[CPU and RAM usage is not showing any values.](https://i.imgur.com/Gdi8s4d.png)

[Time of server is running not showing his real time.](https://i.imgur.com/DfNw07d.png)

**Comment by Ferks-FK** [comment]
This is not a script problem, but something related to a docker/ubuntu update.
This is solved here: https://github.com/pterodactyl/panel/issues/3999

**Reply by RayCodeCSC** [reply]
Not working for me. After restarting VPS doesn't start again.

**Reply by Ferks-FK** [reply]
> Not working for me. After restarting VPS doesn't start again.

Doesn't start again what?

**Reply by RayCodeCSC** [reply]
After changing the grub, and then restarting my VPS, my VPS not working.

**Comment by RayCodeCSC** [comment]
Can i beg author of this script to fix this?

**Reply by vilhelmprytz** [reply]
The author of the script? As mentioned by @Ferks-FK, this is most likely not a problem with the script.

**Reply by RayCodeCSC** [reply]
Yes, now it's works :D . Thanks :)
